sample-notificator-domain/pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <artifactId>sample-notificator</artifactId>
     <groupId>life.qbic</groupId>
-    <version>1.0.0</version>
+    <version>1.0.1</version>
```

Comment:
I am not sure if this is a patch fix. Since functionality is changed (-> project title not mandatory anymore) I think it should be a minor version increase (1.1.0)

Reply:
As you did not really add new functionality but extended existing, I think both would work. As this goes into development, I think you cannot really make it a hotfix (as they are branched off of master)
I am unsure if the feature branch is the right place to increase the version. Should be done on either the development or the master branch IMHO

Reply:
As i specified in the PR Description the Version increase is due to the hotfix #35 (Concerning the exception creation if a project title is missing) that was merged to master but not synced to dev.